velero/spectrum-fusion/recipes/no-olm/core/child-cs-odlm-chart-recipe.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: spp-data-protection.isf.ibm.com/v1alpha1
+kind: Recipe
+metadata:
+  labels:
+    dp.isf.ibm.com/parent-recipe: cpfs-parent-recipe
+    dp.isf.ibm.com/parent-recipe-namespace: <parent recipe namespace>
+  name: cs-odlm-child-recipe
+  namespace: <child recipe namespace>
+spec:
+  appType: common-service
+  groups:
+    - includeClusterResources: true
+      includedResourceTypes:
+        - clusterrole
+        - clusterrolebinding
+        - customresourcedefinitions.apiextensions.k8s.io
+      labelSelector: foundationservices.cloudpak.ibm.com=cs-cluster
+      name: cs-cluster-resources
+      type: resource
+    - includeClusterResources: true
+      includedResourceTypes:
+        # - clusterrole
+        # - clusterrolebinding
+        - customresourcedefinitions.apiextensions.k8s.io
+      labelSelector: foundationservices.cloudpak.ibm.com=odlm-cluster
+      name: odlm-cluster-resources
+      type: resource
+    - includedResourceTypes:
+        - role
+        - rolebinding
+        - serviceaccount
+        - deployments
+        - secrets
+        - commonservices.operator.ibm.com
+      labelSelector: foundationservices.cloudpak.ibm.com=cs-chart
```

Comment:
common service cr is still labeled as commonservice and not included in cs-chart based on the label-common-service.sh script so the cs cr would not be covered in this group.

Reply:
Ok I must not have used your version of the labeling script then, my bad. Only thing is if we change the labeling script we will need to update the no olm br instructions since it looks like the use the existing restore crd and restore cs cr steps https://github.com/IBM/ibm-common-service-operator/pull/2394/#issuecomment-2711539094 (from this issue https://github.ibm.com/IBMPrivateCloud/roadmap/issues/66140). It will probably be easier to leave the labels as is and add additional steps into the restore recipe. This way we also guarantee that the cs cr is there before the operator